tests/console/vmstat.pm

```diff
@@ -0,0 +1,92 @@
+# SUSE's openQA tests
+#
+# Copyright © 2021 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Package: vmstat
+# Summary: Test vmstat installation and verify vmstat and compare the output with/without load.
+# - Install procps
+# - Run vmstat with options
+# - Run vmstat without load and Read output
+# - Run vmstat with load and Read output
+# - Compare the output of vmstat with and without load
+# Maintainer: QE Core <qe-core@suse.de> 
+
+use base 'opensusebasetest';
+use testapi;
+use utils;
+use strict;
+use warnings;
+use POSIX;
+use version_utils;
```

Comment:
I don't think you use that anymore, do you?

Reply:
I have removed "version_utils " which I dont use.